.github/pull_request_template.md

```diff
@@ -1,21 +1,27 @@
-<!-- Thank you for submitting a pull request! --> 
-<!-- Provide an overview of the PR for maintainers and reviewers. -->
+<!-- Thank you for submitting a pull request! Find more information
+at https://trino.io/development/process.html,
+at https://trinodb.github.io/trino-gateway/development/#contributing
+and contact us on #trino-gateway-dev in Slack. -->
+<!-- Provide an overview for maintainers and reviewers. -->
 ## Description
 
 
 
-<!-- Provide details that would help an engineer who is unfamiliar with this part of the code. -->
+<!-- Provide details that help an engineer who is unfamiliar with this part of the code. -->
 ## Additional context and related issues
 
 
 
-<!-- Mark the appropriate option with an (x). Propose a release note if you can. -->
+<!-- Mark the appropriate option with an (x). Propose a release note if you can.
+More info at https://trino.io/development/process#release-note -->
 ## Release notes
 
-( ) This is not user-visible or is docs only, and no release notes are required.
-( ) Release notes are required. Please propose a release note for me.
-( ) Release notes are required, with the following suggested text:
+- ( ) This is not user-visible or is docs only, and no release notes are 
+  required.
+- ( ) Release notes are required. Please propose a release note for me.
+- ( ) Release notes are required, with the following suggested text:
 
 ```markdown
-* 
+## Section
+* Fix some things. ({issue}`issuenumber`)
```

Comment:
This format doesn't work well in this project. The existing release note format is 
```
* xxx. ([#prnumber](https://github.com/trinodb/trino-gateway/pull/prnumber))
or
* xxx. ([#issuenumber](https://github.com/trinodb/trino-gateway/issue/issuenumber))
```